docs/book/src/tasks/installation.md

```diff
@@ -171,6 +171,33 @@ Check the [Docker provider releases](https://github.com/kubernetes-sigs/cluster-
 kubectl create -f {{#releaselink gomodule:"sigs.k8s.io/cluster-api-provider-docker" asset:"provider-components.yaml" version:"0.2.x"}}
 ```
 
+{{#/tab }}
+{{#tab GCP}}
+
+<aside class="note warning">
+
+<h1>Action Required</h1>
+
+Update the path to your GCP credentials file below.
+
+</aside>
+
+##### Create the components
+
+Check the [GCP provider releases](https://github.com/kubernetes-sigs/cluster-api-provider-gcp/releases) for an up-to-date components file.
+
+```bash
+# Create the base64 encoded credentials by catting your credentials json.
+# This command uses your environment variables and encodes
+# them in a value to be stored in a Kubernetes Secret.
+export GCP_B64ENCODED_CREDENTIALS=$( cat /path/to/gcp-credentials.json | base64 | tr -d '\n' )
+
+# Create the components.
+curl -L {{#releaselink gomodule:"sigs.k8s.io/cluster-api-provider-gcp" asset:"infrastructure-components.yaml" version:"0.2.x"}} \
```

Comment:
Unfortunately there is no release of CAPG that exposes these components, we might have to make folks use `make generate-examples` instead

Reply:
What makes the most sense for now? Should we update the version above and merge the doc as-is, but not backport to release-0.2 (with the assumption that the v0.3 release of capi will publish these docs and they'll be working by that point)? Or should I update to use the make command so we can backport?

Reply:
Let's target v1alpha3 only for CAPG, given that we're not planning to do any v1alpha2 release there. We can leave 0.3.x, but the link might be broken until we release, which is probably ok